README.md

```diff
@@ -16,41 +16,118 @@ The purpose of the framework is to provide a simple in app solution for player c
 * 	Seek to a certain offset method
 *	Play in background and control from springboard when in sleep state
 
-#Installation
+## Requirements
 
-iOSAudioPlayer framework works with iOS 8.0 or later. Make sure that the framework is added to 'Embedded Binaries' list. After this, you just import the framework in the file where you will use it.
+- iOS 8.0+
+- Xcode 8.2+
 
-#Usage
-1. Import iOSAudioPlayer Framework
-2. Use shared instance of TPGAudioPlayer class, by calling TPGAudioPlayer.sharedInstance().
-4. Play a certain media file by calling method:
-*		public func playPauseMediaFile(audioUrl: NSURL, springboardInfo: [String : AnyObject], startTime: Double, completion: (previousItem: String?, stopTime: Double) -> ()) 
+## Communication
 
-5. Skip a certain time interval using method:
-*		public func skipDirection(skipDirection: iOSAudioPlayer.SkipDirection, timeInterval: Double, offset: Double)
+- If you **found a bug**, open an issue [github](https://github.com/3pillarlabs/ios-audio-player/issues/new).
+- If you **have a feature request**, open an issue [github](https://github.com/3pillarlabs/ios-audio-player/issues/new).
+- If you **want to contribute**, submit a pull request [github](https://github.com/3pillarlabs/ios-audio-player/compare).
```

Comment:
Communication section should be added before License or About this project section.